app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -55,7 +55,7 @@ class BubbleBuilder extends ConsumerWidget {
             : SwipeTo(
                 onLeftSwipe: !isAuthor
                     ? null
-                    : () {
+                    : (DragUpdateDetails details) {
```

Comment:
Why do we need this arg here? I don't see it being used below...

Reply:
Oh, my bad.
It was needed in my flutter version.
```
Flutter 3.16.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 9e1c857886 (12 days ago) • 2023-11-30 11:51:18 -0600
Engine • revision cf7a9d0800
Tools • Dart 3.2.2 • DevTools 2.28.3
```
Will remove it.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1175/commits/09ee52bb2265d57abf9dc60bda2af49e3b5822fd

Reply:
Sorry, it is needed again.
https://github.com/acterglobal/a3/actions/runs/7184888214/job/19566979989?pr=1175